test/e2e/traceflow_test.go

```diff
@@ -2237,6 +2197,88 @@ func testTraceflowEgress(t *testing.T, data *TestData) {
 	})
 }
 
+func testTraceflowValidation(t *testing.T, data *TestData) {
+	createTestAgnhostPodsHostNetwork := func(tb testing.TB, data *TestData, num int, ns string, nodeName string) ([]string, []*PodIPs, func()) {
+		return createTestPods(tb, data, num, ns, nodeName, true, data.createAgnhostPodOnNode)
+	}
+	podNames, podIPs, cleanupFn := createTestAgnhostPodsHostNetwork(t, data, 1, data.testNamespace, nodeName(0))
```

Comment:
The closure doesn't seem very useful. It could have been one line code but uses 4 lines just to move some parameters to function name.